### DISCUSSION TITLE
Make it possible to search by Lyrics Language

### DISCUSSION THREAD

**Discussion by LouisLouiseKay** [funmusicplace/mirlo#624] [Ideas]
This could simply be done through the tagging system. Once we've added lyrics, there could just be a field saying "indicate lyrics language(s)." maybe with a "?" pop up to explain what that's for.

**Comment by Cwpute** [comment]
This calls for another feature then: being able to tag if there are lyrics/spoken words or not in your releases.
Or maybe you could tag your lyrics' language as being `no lyrics` instead 😄
